Makefile

```diff
@@ -30,27 +30,16 @@ boogie: ${DIR}/boogie/Binaries/Boogie.exe
 tests:
 	(cd "${DIR}"; dotnet test Source/IntegrationTests)
 
-# make test name=<part of the path of an integration test>
+# make test name=<integration test filter>
```

Comment:
These changes to the makefile make it much much easier to test single files without having to rebuild, and also to update individual outputs without having to run the IDE or a complex command (or Lit, which we are not always supporting)